docs/examples/userguide/parallelism/breaking_loop.py

```diff
@@ -0,0 +1,15 @@
+from cython.parallel import prange
```

Comment:
I think you've forgotten to check breaking_loop.pyx into git

Reply:
breaking_loop.pyx is already in GIT https://github.com/cython/cython/blob/master/docs/examples/userguide/parallelism/breaking_loop.pyx, there are no changes needed there so it is not included in PR.

Reply:
Sorry you're quite right. Clearly I wasn't being very clever earlier...

Reply:
But now I see that at least .pyx file needs alignment. I will fix it now.

Reply:
alignment fixed.